lib/linguist/languages.yml

```
@@ -7976,3 +7976,10 @@ xBase:
   tm_scope: source.harbour
   ace_mode: text
   language_id: 421
+yuescript:
```

This should be capitalised as it is in the actual language name.

```suggestion
Yuescript:
```